locale/en/manageiq.po

```diff
@@ -41798,6 +41883,314 @@ msgstr ""
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::PricePlan|Region number"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::PricePlan|Resource type"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Calculation"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Concept"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Entity"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Field"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Group"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Href slug"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Region description"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Region number"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Screener"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Start date"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Tier input variable"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Tiers use full value"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Rate|Uses single tier"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Fixed rate currency"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Fixed rate per time"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Fixed rate subunits"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Href slug"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Region description"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Region number"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Step time unit"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Step time value"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Step unit"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Step value"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Tier end value"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Tier start value"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Variable rate currency"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Variable rate per time"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Variable rate per unit"
+msgstr ""
+
+#: ../config/model_attributes.rb
+msgid "ManageIQ::Showback::Tier|Variable rate subunits"
```

Comment:
I have on idea why showback is starting to show up here

Reply:
Yeah really strange. 🤷

Reply:
So far, it looks like we're not excluding this on master but are on quinteros.  Perhaps from [here](https://github.com/ManageIQ/manageiq/blob/f75a6667a530f6ba73fbb74bf388cef425f7e761/lib/tasks/model_attribute_override.rb#L7-L12).  Looking to see if we're pulling in new gettext i18 on master...

Reply:
master: 

![image](https://github.com/ManageIQ/manageiq/assets/19339/8c56fab0-8fc8-4ac1-8b7c-a16a0b96d359)


looks like gettext gems are the same

Reply:
Note, it's the same table and columns on quinteros

Reply:
Ok, it looks like the difference is master does eager load and these classes are defined, whereas they're not on quinteros.

```
irb(main):002:0>  Rails.application.eager_load!
PostgreSQLAdapter#log_after_checkout, connection_pool: size: 15, connections: 1, in use: 1, waiting_in_queue: 0
=> nil
irb(main):003:0> defined?(ManageIQ::Showback::Tier)
=> "constant"
```

Reply:
on quinteros:

```
irb(main):002:0>  Rails.application.eager_load!
=> []
irb(main):003:0> defined?(ManageIQ::Showback::Tier)
=> nil
```

Reply:
I guess this is ok.  I'm not sure if it was incorrectly being excluded before.  I also don't know why the fully namespaced class name is there.  🤷   @Fryguy do you have any idea?

Reply:
I bet it's because of zeitwerk, and the fact that the gem name is manageiq-consumption and the classes are ManageIQ::Showback.  That tells me there's some "special" handling there for the zeitwerk stuff.

Reply:
> I bet it's because of zeitwerk, and the fact that the gem name is manageiq-consumption and the classes are ManageIQ::Showback. That tells me there's some "special" handling there for the zeitwerk stuff.

Yes, that's a great insight.  I tried to narrow it down but got stuck.  It's quite fortunate we've had no new message catalogs since zeitwerk was enabled so what we're seeing here is really due to zeitwerk or add/changed/removed messages.  I'll have to try again.

Reply:
Ok, so, looking at this further, zeitwerk knows how to eager load the plugin and it's the only plugin I see with namespacing and it's own tables, which is why it's different.  The classic loader doesn't eager load it because it doesn't know how to walk the paths, so that's why we didn't see it previously.  I would say it was a bug that it was excluded.  I think this change is fine.